ui/dashboard/src/@locales/ja/form.json

```
@@ -354,5 +354,18 @@
   "name-email-search-placeholder": "名前、メールアドレスを検索...",
   "name-desc-search-placeholder": "名前、説明を検索...",
   "email-search-placeholder": "メールアドレスを検索...",
-  "name-search-placeholder": "名前を検索..."
+  "name-search-placeholder": "名前を検索...",
+  "discard-form:discard-rule.discard-unsaved-changes": "未保存の変更を破棄しますか？",
+  "prerequisite-discard-desc": "前提フラグ <b>{{flagName}}</b>（バリエーション付き）を{{action}}",
+  "individual-discard-desc": "{{action}} ユーザーに対して <variantElement /> に <b>{{flagName}}</b> を設定",
+  "custom-rule-discard-desc": "{{action}} 条件を",
+  "custom-rule-clause-value-discard-desc": "{{action}} 条件から値 <b>{{value}}</b> を",
+  "custom-rule-strategy-discard-desc": "{{action}} ルールのバリエーションを",
+  "custom-rule-strategy-add-new-discard-desc": "ルール {{clauseLabel}} を追加して配信",
+  "custom-rule-reorder-discard-desc": "ルールを並べ替える",
+  "custom-rule-audience-discard-desc": "{{action}} オーディエンスのトラフィック配分を",
+  "custom-rule-audience-not-include-desc": "<p>この配分に含まれていない <b>{{percent}}%</b> には <b>{{variation}}</b> が提供されます</p>",
+  "custom-rule-audience-include-desc": "<p>この配分に含まれている <b>{{percent}}%</b> には提供されます</p>",
+  "custom-rule-default-strategy-discard-desc": "デフォルトルールに設定",
```

for consistency with others
```suggestion
  "custom-rule-default-strategy-discard-desc": "デフォルトルールに設定します",
```